src/components/VmDetails/CardEditButton.js

```diff
@@ -44,9 +46,11 @@ class CardEditButton extends React.Component {
     const myClick = editEnabled ? noop : this.enableEditHandler
 
     return (
-      <a title={tooltip} onClick={(e) => { e.preventDefault(); myClick() }} className={classes} id={id}>
-        <Icon type='pf' name='edit' />
-      </a>
+      <OverlayTrigger overlay={<Tooltip id={`${id}-edit-tooltip`}>{tooltip}</Tooltip>} placement='bottom' id={id} onClick={(e) => { e.preventDefault(); myClick() }} >
```

Comment:
- `id={id}` should be in `a`. 
 - I think it is a little confusing when `onClick` in `OverlayTrigger`, it would be better to move that prop to `a` as well.
 - `<Tooltip id={'${id}-edit-tooltip'}>{tooltip}</Tooltip>` -> `<Tooltip id={'${id}-tooltip'}>{tooltip}</Tooltip>`, because `id` already contains `-edit`